src/modules/ipfs/ipfs.controller.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  Body,
+  Controller,
+  Get,
+  Param,
+  Post,
+  UseInterceptors,
+} from '@nestjs/common';
+import { ApiBody, ApiOperation, ApiParam, ApiTags } from '@nestjs/swagger';
+import { CID } from 'multiformats/cid';
+import { CIDPipe } from '../../common/cid.pipe';
+import { Auth } from '../auth/auth.decorator';
+import { SentryErrorInterceptor } from '../interceptors/sentry-error-interceptor';
+import { IPFSService } from './ipfs.service';
+
+@Auth()
+@UseInterceptors(SentryErrorInterceptor)
+@Controller({ path: 'ipfs', version: '1' })
+export class IPFSController {
+  constructor(private ipfsService: IPFSService) {}
+
+  @Get('/:cid')
+  @ApiTags('IPFS')
+  @ApiOperation({
+    summary: 'Returns credential from IPFS Store',
+    description: 'Returns credential represented by service in DID document.',
```

Comment:
I think it's better to have this more generic. It's not really specific to credentials, so no need to limit it to that, even if that is what we are currently using it for.
```suggestion
    summary: 'Returns content from IPFS Store',
    description: 'Returns data identified by the provided content identifier (cid). This can be used, for example, to resolve credentials linked in the service endpoint of a DID document.',
```